dask/base.py

```diff
@@ -416,14 +416,32 @@ def tokenize(*args, **kwargs):
     return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
 
 
+def dont_optimize(dsk, keys):
+    return dsk
+
+
+def optimization_function(obj):
```

Comment:
Actually, I think I like the decorator method the best. Compatible with existing code and explicit about the option name.